fxgl/src/main/kotlin/com/almasb/fxgl/app/scene/FXGLDefaultMenu.kt

```diff
@@ -224,11 +230,11 @@ open class FXGLDefaultMenu(type: MenuType) : FXGLMenu(type) {
         emitter.setSpawnPointFunction { Point2D.ZERO }
         emitter.setAccelerationFunction { Point2D(random(-1.0, 1.0), random(0.0, 0.0)) }
 
-        val box = HBox(text, text2)
+        val box = HBox(firstLetter, restOfTheTitle)
         box.alignment = Pos.CENTER
 
         val titleRoot = StackPane(border, box)
-        titleRoot.translateX = appWidth / 2.0 - (textWidth + 30) / 2
+        titleRoot.translateX = appWidth / 2.0 - (titleWidth + 30) / 2
```

Comment:
The translateX property should also be bound to the new value of `restOfTheTitle` layout bounds. Something like:

```
firstLetter.layoutBoundsProperty().addListener { _, _, newBounds -> update titleRoot translateX }
```